lib/ldp/client/methods.rb

```diff
@@ -8,11 +8,24 @@ module Ldp::Client::Methods
   def initialize_http_client *http_client
```

Comment:
I'm fine with either - but we still need LDP not to unconditionally overwrite HTTP headers.